Formula/git-cal.rb

```diff
@@ -4,8 +4,14 @@ class GitCal < Formula
   url "https://github.com/k4rthik/git-cal/archive/v0.9.1.tar.gz"
   sha256 "783fa73197b349a51d90670480a750b063c97e5779a5231fe046315af0a946cd"
   license "MIT"
+  revision 1
   head "https://github.com/k4rthik/git-cal.git"
 
+  livecheck do
+    url :head
```

Comment:
same thing, :stable is preferred over :head, we can followup in the next PR.

cc @samford